src/test/integration/IntegrationBase.t.sol

```diff
@@ -101,7 +101,10 @@ abstract contract IntegrationBase is IntegrationDeployer {
 
             operator.registerAsOperator();
             operator.depositIntoEigenlayer(strategies, tokenBalances);
-        }
+
+            // Roll passed the allocation configuration delay
+            rollForward({blocks: block.number + ALLOCATION_CONFIGURATION_DELAY});
```

Comment:
Incorrect use of `rollForward`, since it rolls forward (from the current block number) by `blocks`.

Should be `rollForward({blocks: ALLOCATION_CONFIGURATION_DELAY});`.